python/fledge/plugins/south/mqtt-readings/mqtt-readings.py

```diff
@@ -130,11 +130,18 @@
         'minimum': '0',
         'maximum': '2'
     },
+    'json_object_key': {
+        'description': 'If only a value is sent this needs to be converted to an json object, the value given here will he the key for the value in that json object',
+        'type': 'string',
+        'default': 'value',
+        'order': '8',
+        'displayName': 'Json Object Key',
+    },
     'assetName': {
         'description': 'Name of Asset',
         'type': 'string',
         'default': 'mqtt-',
-        'order': '8',
+        'order': '9',
```

Comment:
I would suggest to keep this one level up, i.e. above the datapoint name. This is, to which the reading object maps.

Ideally, In future, We can think of an option to allow mapping asset name per topic.